lib/kast-symbolic.kscript

```diff
@@ -5,7 +5,7 @@ alias ktry x      = try x    | kast -i kore -o pretty -d .build/defn/prove /dev/
 alias ktryf x     = tryf x   | kast -i kore -o pretty -d .build/defn/prove /dev/stdin
```

Comment:
Delete this file, you have a guard against executing the REPL with `$backend = symbolic` in the `kmich` script.